docs/source/releases/v1_14_0a0.rst

```diff
@@ -0,0 +1,31 @@
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
```

Comment:
I hope you don't mind if I start updating these one off... I'll do a wholesale change at some point, but now they bug me :-)